Add draft landing page with topic cards (#104)
cosmos/docs: docs/cosmos-stack-overview.mdx

---
title: "Cosmos Stack Developer Documentation"

> description: "The most widely-adopted and battle-tested Layer 1 blockchain technology stack, in production with 200+ chains today"

Reviewer: ```suggestion
description: "The most widely-adopted and battle-tested Layer 1 blockchain technology stack, in production with 200+ chains today."
```

Author: I propose an alternative (see latest)

      color: '#cccccc',
      lineHeight: '1.6',
    }}>
      The most widely-adopted and battle-tested Layer 1 blockchain technology stack, in production with 200+ chains today
    </p>
  </div>

  {/* Content */}
  <div style={{
    maxWidth: '1200px',
    margin: '0 auto',
    padding: '60px 40px',
  }}>
    {/* Introduction */}
    <section style={{ marginBottom: '80px', paddingBottom: '80px', borderBottom: '1px solid #cccccc10' }}>
      <h2 style={{
        fontSize: '32px',
        fontWeight: '700',
        marginBottom: '20px',
        color: '#F1F1F1',
      }}>
        Introduction
      </h2>
      <p style={{
        fontSize: '18px',
        lineHeight: '1.8',
        color: '#cccccc',
        marginBottom: '16px',
      }}>
        The Cosmos stack is the most widely-adopted and battle-tested Layer 1 blockchain technology stack, in production with <strong>200+ chains</strong> today. It is performant, customizable, and EVM-compatible, offering engineers full end-to-end control of their blockchain infrastructure and implementation.
      </p>
      <p style={{
        fontSize: '18px',
        lineHeight: '1.8',
        color: '#cccccc',
      }}>

>         The stack is modular by design—use pre-built code modules or integrate custom features and logic for your use case, compliance needs, governance, consensus, and performance requirements. This documentation explains the open-source stack components: <strong>Cosmos SDK</strong>, <strong>CometBFT</strong>, <strong>IBC</strong>, and <strong>Cosmos EVM</strong>.

Reviewer: ```suggestion
        The stack is modular by design — leverage pre-existing core components and integrate custom features and logic for your use case, compliance needs, governance, consensus, and performance requirements. This documentation explains the open-source stack components: <strong>Cosmos SDK</strong>, <strong>CometBFT</strong>, <strong>IBC</strong>, and <strong>Cosmos EVM</strong>.
```

>         The stack is modular by design—use pre-built code modules or integrate custom features and logic for your use case, compliance needs, governance, consensus, and performance requirements. This documentation explains the open-source stack components: <strong>Cosmos SDK</strong>, <strong>CometBFT</strong>, <strong>IBC</strong>, and <strong>Cosmos EVM</strong>.

Reviewer: I would also order it same as: https://github.com/cosmos/docs/pull/104/files#r2446621701

      </p>
    </section>

    {/* Stack Components */}
    <section style={{ marginBottom: '80px', paddingBottom: '80px', borderBottom: '1px solid #cccccc10' }}>
      <h2 style={{
        fontSize: '32px',
        fontWeight: '700',
        marginBottom: '20px',
        color: '#F1F1F1',
      }}>

>         Get Started with Cosmos SDK

Reviewer: ```suggestion
        Get Started with the Cosmos Stack
```

      </h2>
      <p style={{
        fontSize: '18px',
        lineHeight: '1.8',
        color: '#cccccc',
        marginBottom: '32px',
      }}>
        Explore the modular components that make up the Cosmos stack. Each component is optional, provides specific functionality and can be customized for your use case.
      </p>

      <TopicCardGrid cols={3}>
        <TopicCard

>           title="CometBFT"

Reviewer: I would do Cosmos SDK first, then Cosmos EVM, then IBC, and then CometBFT.

          ]}
        />

        <TopicCard
          title="Getting Started"
          description="Quick start guides and tutorials to begin building on the Cosmos stack"
          color="#666666"
          links={[
            { text: 'Quick Start', href: '/docs/evm/next/documentation/getting-started/index' },
            { text: 'Local Node Setup', href: '/docs/evm/next/documentation/getting-started/local-node-setup' },
            { text: 'Development Environment', href: '/docs/evm/next/documentation/getting-started/tooling-and-resources/development-environment' },
            { text: 'FAQ', href: '/docs/evm/next/documentation/getting-started/faq' },
          ]}
        />
      </TopicCardGrid>
    </section>

    {/* About Sections */}
    <section style={{ marginBottom: '80px', paddingBottom: '80px', borderBottom: '1px solid #cccccc10' }}>

      <div style={{
        background: '#0E0E0E',
        border: '1px solid #cccccc20',
        borderRadius: '12px',
        padding: '40px',
        marginBottom: '24px',
      }}>
        <h3 style={{
          fontSize: '24px',
          fontWeight: '600',
          marginBottom: '16px',
          color: '#F1F1F1',
        }}>

>           About CometBFT

Reviewer: Can we make each "About" component collapsable? Where you expand to see the text? (sounds like bad design tho..)

Can you order Cosmos SDK, Cosmos EVM, IBC, CometBFT

Reviewer: Repetitive About on each feels bad, maybe the section title can be About and then each card just has the product name in it

Author: yeah I think there is a better element we could use for these im just not completely sure what it would be. The idea (not my idea) was to keep this very simple like some specific old Linux webpage from 2001 lol

          fontWeight: '600',
          marginBottom: '16px',
          color: '#F1F1F1',
        }}>
          About Cosmos EVM
        </h3>
        <p style={{
          fontSize: '16px',
          lineHeight: '1.7',
          color: '#cccccc',
        }}>
          Cosmos EVM is a plug-and-play solution that adds EVM compatibility and customizability to Cosmos SDK chains. It provides complete Ethereum capabilities: Solidity smart contracts, Ethereum JSON-RPC, native support for the EVM wallet/token/user experience, and access to the entire Ethereum developer ecosystem. Its <strong>precompiles and extensions</strong> allow developers to leverage modules like IBC within the EVM and get native ERC-20 support for tokens on Cosmos. It can run any valid smart contract from Ethereum and also implement new features that are not yet available on the standard Ethereum VM.
        </p>
      </div>
    </section>

    {/* FAQ */}
    <section style={{ marginBottom: '80px', paddingBottom: '80px', borderBottom: '1px solid #cccccc10' }}>
      <h2 style={{
        fontSize: '32px',
        fontWeight: '700',
        marginBottom: '32px',
        color: '#F1F1F1',
      }}>

>         Frequently Asked Questions

Reviewer: I would remove for now
